### DISCUSSION TITLE
Custom dictionaries order for a book

### DISCUSSION THREAD

**Discussion by ichnilatis-gr** [koreader/koreader#10883] [Q&A] [answered]
There used to be a feature where you could long tap on the title of a dictionary to make it the primary dictionary for a particular book. Doesn't this feature exist anymore?

**Comment by poire-z** [comment, marked as answer]
It is **short** tap on the dict window title - and the title is then prefixed with `(1)`.

**Reply by ichnilatis-gr** [reply]
Thank you! I though I had tried it, but I was wrong.
